src/Shared/Recruit.Vacancies.Client/Infrastructure/ReferenceData/BankHolidays/BankHolidayProvider.cs

```diff
@@ -36,5 +33,10 @@ public async Task<List<DateTime>> GetBankHolidaysAsync()
                         .ToList();
                 });
         }
+
+        public async Task<BankHolidays> GetBankHolidayListAsync()
```

Comment:
I'd delete this from here and just talk directly to referenceDateReader from BankHolidayUpdateService